packages/builder/src/components/automation/AutomationBuilder/FlowChart/AutomationBlockTagline.svelte

```diff
@@ -60,6 +60,7 @@
 </script>
 
 <div>
+  <!-- eslint-disable-next-line svelte/no-at-html-tags-->
```

Comment:
There's various places we unsafely inject HTML, but given what our product is that's not really avoidable, so just comment these errors out.